webpack/components/NewTemplateSync/components/ProxySettingFields.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field as FormikField } from 'formik';
+import { translate as __ } from 'foremanReact/common/I18n';
+import CommonForm from 'foremanReact/components/common/forms/CommonForm';
+import { InputGroup } from 'patternfly-react';
+import { FieldLevelHelp } from 'patternfly-react/dist/js/components/FieldLevelHelp';
+
+import {
+  tooltipContent,
+  label,
+} from './NewTemplateSyncForm/NewTemplateSyncFormHelpers';
+import ProxySettingField from './ProxySettingField';
+
+const ProxySettingsFields = ({
+  proxyPolicySetting,
+  proxyIdSetting,
+  syncType,
+  resetField,
+  formProps: { isSubmitting },
+}) => {
+  if (Object.keys(proxyPolicySetting).length === 0) {
+    return <></>;
+  }
+  const proxyPolicyFieldName = `${syncType}.http_proxy_policy`;
+  const proxyIdFieldName = `${syncType}.http_proxy_id`;
+
+  const descriptionEdits = {
+    '? ': '?<br>',
+    'Custom HTTP proxy': '<b>Custom HTTP proxy</b>',
+    ' when syncing templates': '',
+  };
+  proxyPolicySetting = proxyPolicySetting.set(
+    'description',
+    `${proxyPolicySetting.description.replace(
+      /\b(\? |Custom HTTP proxy| when syncing templates)\b/g,
+      match => descriptionEdits[match]
+    )}`
+  );
```

Comment:
This causes the description not to be translated.
`N_('Should an HTTP proxy be used for template sync? If you select Custom HTTP proxy, you will be prompted to select one when syncing templates.')` - this marks it to be translated in the future, for example in the js code with `__(proxyPolicySetting.description)`.
Maybe it will be better to use the <br> in lib/foreman_templates/engine.rb but I'm not sure if the description is used anywhere else

Reply:
1. yes, I didnt check the code before, but it looks like modifyDescription will not be translated.
2. We can add new strings in the UI, that mark for translated is used so we can send variables to be translated:
`translate_later = N('string')` and then `__(translate_later)`

Reply:
I see, thank you for the explanation. In that case, I will not modify the proxy settings descriptions in the UI as I feel like it would unnecessarily complicate the code.